database_path: "~/.config/niryo"